backend/validation/signUp.js

```diff
@@ -1,7 +1,34 @@
 const Validator = require("validator");
 const isEmpty = require("is-empty");
+const keys = require("../config/keys");
 
-module.exports = function validateRegisterInput(data) {
+const axios = require("axios");
+
+const apiKey = keys.apiKey;
+
+async function getUserByEmail(email) {
+  try {
+    const response = await axios.post('https://us-east-2.aws.data.mongodb-api.com/app/data-iudir/endpoint/data/v1/action/findOne', {
```

Comment:
seems like we have the same base URL for the MongoDB source hard-coded in quite a few places.  if we ever needed to change that (for example we find us-east-2 of AWS is no longer available and we have to switch to us-central or something) it would create a lot of work to change every instance of this ... how about setting it once in a `const` somewhere and referencing it?

Reply:
Ok I will do that! Thanks for pointing that out.

Reply:
Oh yeah I actually did have it in the keys.js file I just forgot about it. I will be implementing that now.